lib/src/backend.rs

```diff
@@ -201,6 +202,33 @@ impl MergedTreeId {
             MergedTreeId::Merge(tree_ids) => tree_ids.clone(),
         }
     }
+
+    /// Represent a `MergeTreeId` in a user friendly way. This makes no
+    /// stability guarantee, as the format may change at any time.
+    #[allow(clippy::inherent_to_string)]
+    pub fn to_string(&self) -> String {
```

Comment:
I think it's better to call it something else instead of allowing the clippy lint. But also agree with Yuya that we probably don't need this method at all, so...

Reply:
Renamed for now.

Reply:
I still think it's better to implement this in the module where it's used, at least until we see that there's a common need for it.

Reply:
I would propose to move the conversation to #2686 and closing this pr, after implementing the above.